client/src/components/rightbar/Rightbar.jsx

```diff
@@ -6,21 +6,20 @@ import { useDispatch, useSelector } from 'react-redux'
 import { useState } from 'react'
 import EditProfileModal from '../editProfileModal/EditProfileModal'
 import AddIcon from '@mui/icons-material/Add'
-import RemoveIcon from '@mui/icons-material/Remove' 
+import RemoveIcon from '@mui/icons-material/Remove'
 
 const Rightbar = ({ profileUser }) => {
   const user = useSelector((state) => state.authReducer.authData)
-  const [following, setFollowing] = useState(false)
```

Comment:
There's no need for this internal `useState` stuff as the stuff passed into this component via redux will already get updated when things change. So when we call `handleFollow` that ultimately updates the user object, which means the result from this line changes and causes the component to re-render:

https://github.com/jonnyabrams/bookface/blob/5cd74648f7490ef4e4bdb2ac63c5654c0fd63955/client/src/components/rightbar/Rightbar.jsx#L12

And when the component re-renders this line will be re-evaluated and result in the new `following` state:

https://github.com/alexandermendes/bookface/blob/5cd74648f7490ef4e4bdb2ac63c5654c0fd63955/client/src/components/rightbar/Rightbar.jsx#L17

So just removes a bit of complexity to drop the `setFollowing` bit. The less things we can have all updating some state the better!